cmd/mattermost/commands/webhook.go

```diff
@@ -23,6 +24,14 @@ var WebhookListCmd = &cobra.Command{
 	RunE:    listWebhookCmdF,
 }
 
+var WebhookCreateIncomingCmd = &cobra.Command{
+	Use:     "create-incoming",
+	Short:   "Create incoming webhook",
+	Long:    "create incoming webhook",
+	Example: "  webhook create-incoming",
```

Comment:
Would love to see basic example of the command.  It could be like `webhook create-incoming [myteam] --title [webhookTitle] --description [webhookDescription] --channel mychannel --lock-to-channel --icon [iconURL]`

Reply:
Added command example